### DISCUSSION TITLE
[613900]  Museum of  Other  realities

### DISCUSSION THREAD

**Discussion by LW233333333** [pjy612/SteamManifestCache#52] [Share]
"613901"
					{
						"DecryptionKey"		"53cd0c4bfc97eab6b81e3ad8cc2b87f02befdcfee5593170ed6064fe770a8710"
[Uploading 613901_8840624391063568479.zip…]()

**Comment by ZEERDEER** [comment]
文件上传完成后再提交发布。

**Reply by LW233333333** [reply]
如何上传压缩文件，我试了好几次它显示正在上传然后消失了

**Comment by LW233333333** [comment]
[613901_8840624391063568479.zip](https://github.com/pjy612/SteamManifestCache/files/13691441/613901_8840624391063568479.zip)

**Reply by LW233333333** [reply]
竟然可以了

**Comment by LW233333333** [comment]
code：1C8D080A1DF80CEF

**Reply by pjy612** [reply]
查收邮件
